src/colvar.cpp

```diff
@@ -886,7 +886,9 @@ int colvar::init_components(std::string const &conf)
   error_code |= init_components_type<euler_phi>(conf, "euler phi angle of the optimal orientation", "eulerPhi");
   error_code |= init_components_type<euler_psi>(conf, "euler psi angle of the optimal orientation", "eulerPsi");
   error_code |= init_components_type<euler_theta>(conf, "euler theta angle of the optimal orientation", "eulerTheta");
+  #if (__cplusplus >= 201103L)
```

Comment:
Shouldn't this also be wrapped in `#ifdef LEPTON`?

Reply:
I added `#ifdef LEPTON` in the latest commit, although the code can be compiled without it and fallback to `linearCombination`.

Reply:
Good, thanks. We're in a fairly good place now, John says that we can rely on C++11 for new features, as he'll make at least one C++11-enabled VMD build for Linux that people can use. We basically need to make sure that VMD as a whole doesn't break for people on very old machines (past EOL), which by now ought to accept that they'll get reduced functionality. Most VMD users will get the full set of features.

A Lepton dependency remains trickier, as that *will* prevent distribution with VMD.